src/pages/index.tsx

```diff
@@ -10,14 +11,15 @@ const MediumCourt = dynamic(() => import("@/components/MediumCourt"), { ssr: fal
 const SmallCourt = dynamic(() => import("@/components/SmallCourt"), { ssr: false });
 
 const Home: NextPage = () => {
+  const { name: courtName } = useStoreSelector((state) => state.courtName);
   return (
     <HeaderLayout>
-      <SmallCourt />
-      <MediumCourt />
-      <HalfCourt />
-      <ProHalfCourt />
-      <FullCourt />
-      {/* <ProFullCourt /> */}
+      {courtName === "510 m² Pro Court (17 m × 30 m)" && <ProFullCourt />}
+      {courtName === "420 m² Full Court (15 m × 28 m)" && <FullCourt />}
+      {courtName === "210 m² Pro Half Court (15 m × 14 m)" && <ProHalfCourt />}
+      {courtName === "150 m² Half Court (15 m × 10 m)" && <HalfCourt />}
+      {courtName === "70 m² Medium Court (7 m × 10 m)" && <MediumCourt />}
+      {courtName === "45 m² Small Court (5 m × 9 m)" && <SmallCourt />}
```

Comment:
Thank you for your comment. 
We might use court id instead of those complex strings after connect with our DB. At the moment, this is the simplest implementation with minimal changes.